packages/docs/src/components/theme-scales.js

```diff
@@ -44,7 +44,7 @@ export default props => {
                 {table[key].map((property, index) => (
                   <Styled.inlineCode>
                     {!!index && ', '}
-                    {property}
+                    <a href={`https://developer.mozilla.org/en-US/docs/Web/CSS/${property}`}>{property}</a>
```

Comment:
Using the `Styled.a` component should pick up the theme's link files:
```suggestion
                    <Styled.a href={`https://developer.mozilla.org/en-US/docs/Web/CSS/${property}`}>{property}</Styled.a>
```